pallets/subtensor/src/lib.rs

```diff
@@ -1147,15 +1147,21 @@ pub mod pallet {
 
                     Stake::<T>::insert(hotkey.clone(), coldkey.clone(), stake);
 
-                    next_uid += 1;
+                    next_uid = next_uid.checked_add(1).expect(
+                        "should not have total number of hotkey accounts larger than u16::MAX",
+                    );
```

Comment:
u16::MAX is only 65535. is this enough? I think we're close to that, if not already exceeded it?

Reply:
If so, how does this line currently work without my changes?

Reply:
yeah this is fine